### PR TITLE
fix: add `agent_ids` fields to filter and order by agent id when listing sessions

### PR DESCRIPTION
**Pull request by fregataa**
~~blocked by: #1022~~

Enable to list sessions by `agent_id` of sibling kernels

### CODE REVIEW

**Comment by achimnol** [comment]
Now merged #1022!

File: src/ai/backend/client/func/session.py [hunk 1]
```diff
@@ -126,7 +126,8 @@ async def paginated_list(
                 "filter": (filter, "String"),
                 "order": (order, "String"),
             },
-            fields,
+            # fields,
+            [session_fields["session_id"], session_fields["agent_ids"]],
```
Comment (achimnol) [inline src/ai/backend/client/func/session.py]: This removes all the default fields in the `backend.ai ps` command! Please include `agent_ids` inside `_default_list_fields`.

**Review by achimnol — CHANGES_REQUESTED** [review]
Mostly looks good, please check out the following minor fix.

File: src/ai/backend/client/func/session.py (hunk 1 above)
Comment (fregataa) [reply src/ai/backend/client/func/session.py]: thanks! my mistake..

**Review by achimnol — APPROVED** [review]

**Pull request merged**